BibBuddy/app/src/main/java/de/bibbuddy/ExportBibTex.java

```diff
@@ -176,15 +127,16 @@ public String getBibDataFromBook(Long bookId, BookDao bookDao, NoteDao noteDao)
 
   private String getBibKey(Book book) {
     // remove whitespaces from book's title
-    return book.getTitle().replaceAll("\\s+", "") + "," + "\n";
+    return book.getTitle().replaceAll("\\s+",
+        "") + "," + "\n";
```

Comment:
Why did you make a newline here?
```suggestion
    return book.getTitle().replaceAll("\\s+", "") + "," + "\n";
```

Reply:
This is needed for the format of the BibTeX file, after the title of the book, there should be a new line.